src/modelAnalysis/rumba/pValDistForModelOverlap.m

```diff
@@ -0,0 +1,58 @@
+function [rxnsInCommon, MedianChange] = pValDistForModelOverlap(model1, model2)
+% Compute for each reaction in common in both sampled modelw
+% the magnitude of median flux value change
+%
+% USAGE:
+%
+%    [rxnsInCommon, MedianChange] = pValDistForModelOverlap(model1, model2)
+%
+% INPUTS:
+%    model1:          Model sampled under first condition
+%    model2:          Model sampled under second condition
+%
+% OUTPUTS:
+%    rxnsInCommon:    Reactions shared by both sampled models
+%    MedianChange:    Magnitude of median flux value change for
+%                     each reaction listed in 'rxnsInCommon'
+%
+% .. Authors:
+%       - Nathan E. Lewis, May 2010-May 2011
+%       - Anne Richelle, May 2017
+
+ZeroRxns1 = setdiff(model1.rxns, model2.rxns);
+ZeroRxns2 = setdiff(model2.rxns, model1.rxns);
+
+% Add in changed zero flux rxns %%% make sure this doesn't conflict with
+% later steps
+model1.rxns(end+1:end+length(ZeroRxns2))=ZeroRxns2;
+model2.rxns(end+1:end+length(ZeroRxns1))=ZeroRxns1;
+model1.points(end+1:end+length(ZeroRxns2),:)=0;
+model2.points(end+1:end+length(ZeroRxns1),:)=0;
+
+% make a list of reactions that are shared
+rxnsInCommon = intersect(model1.rxns,model2.rxns);
```

Comment:
Yes, you right, but was initiallly put to be sure that the addMissingReactions subroutine has done properly its job... i will add a warning at this step if the intersect is not equal to model1/2.rxns ;)